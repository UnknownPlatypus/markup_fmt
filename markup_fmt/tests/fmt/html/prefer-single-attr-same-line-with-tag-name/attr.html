<div class="app"></div>
<div
class="app"></div>

<div class="app" id=""></div>
<div
class="app" id=""></div>
