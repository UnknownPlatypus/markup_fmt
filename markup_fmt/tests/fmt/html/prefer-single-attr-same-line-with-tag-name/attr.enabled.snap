---
source: markup_fmt/tests/fmt.rs
---
<div class="app"></div>
<div class="app"></div>

<div class="app" id=""></div>
<div
  class="app"
  id=""
>
</div>
